src/rosdoc_lite/landing_page.py

```diff
@@ -82,7 +82,12 @@ def generate_links(package, manifest, rd_configs):
     :param rd_configs: [dict] package manifest rosdoc configs
     :returns: [str] list of html snippets
     """
-    config_list = [c for c in rd_configs.itervalues() if c['builder'] != 'rosmake']
+    try:
+        rd_configs_values = rd_configs.values()
```

Comment:
I believe that we should be able to just use `values()` in both 2.7 and 3.x

http://wiki.ros.org/python_2_and_3_compatible_code